pkg/ami/api.go

```diff
@@ -57,6 +57,24 @@ func IsAvailable(api ec2iface.EC2API, id string) (bool, error) {
 	return *output.Images[0].State == "available", nil
 }
 
+// LookupRootDeviceName gets the root block device mapping
+func LookupRootDeviceName(api ec2iface.EC2API, id string) (string, error) {
+	input := &ec2.DescribeImagesInput{
+		ImageIds: []*string{&id},
+	}
+
+	output, err := api.DescribeImages(input)
+	if err != nil {
+		return "", errors.Wrapf(err, "unable to determine root device name for %q", id)
+	}
+
+	if len(output.Images) < 1 {
```

Comment:
We'd need a check for when it's 0 as well, or use `!= 1` (but will need to adjust the error message too).

Reply:
I suppose that with just the single image ID search parameter, we will only ever get one or none.

Reply:
Would you like me to add the comment to the IsAvailable function as they are now merged.

Reply:
Yeah, please do!

Reply:
I have added the comment